website/redirects.js

```diff
@@ -0,0 +1,1234 @@
+  //   destination: 'https://github.com/hashicorp/nomad-spark',
+  //   permanent: true,
+  // },
+
+  {
+    source: '/guides/operating-a-job/(index.html)?',
+    destination: 'https://learn.hashicorp.com/collections/nomad/manage-jobs',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/accessing-logs(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/jobs-accessing-logs',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/configuring-tasks(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-configuring',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/external/(index.html)?',
+    destination: 'https://learn.hashicorp.com/collections/nomad/plugins',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/external/lxc(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/plugin-lxc',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/inspecting-state(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-inspec',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/resource-utilization(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-utilization',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/submitting-jobs(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/jobs-submit',
+    permanent: true,
+  },
+
+  {
+    source:
+      '/guides/operating-a-job/advanced-scheduling/advanced-scheduling(.html)?',
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/advanced-scheduling',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/advanced-scheduling/affinity(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/affinity',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/advanced-scheduling/preemption-service-batch(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/preemption',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/advanced-scheduling/spread(.html)?',
+    destination: 'https://learn.hashicorp.com/tutorials/nomad/spread',
+    permanent: true,
+  },
+
+  {
+    source: '/guides/operating-a-job/failure-handling-strategies',
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/job-failure-handling',
+    permanent: true,
+  },
+  {
+    source:
+      '/guides/operating-a-job/failure-handling-strategies/check-restart(.html)?',
+    destination:
+      'https://learn.hashicorp.com/tutorials/nomad/failures-check-restart',
+    permanent: true,
+  },
+  {
+    source: '/guides/operating-a-job/failure-handling-strategies/index.html',
+    destination:
+      'https://learn.hashicorp.com/collections/nomad/job-failure-handling',
+    permanent: true,
+  },
```

Comment:
This can be combined with the redirect two above by making `index` optional i think: